app/views/donations/new.html.haml

```diff
@@ -1,11 +1,8 @@
-- content_for :head do
-  %script{:type => "text/javascript", :src => "https://checkout.stripe.com/checkout.js"}
-
+- content_for :page_footer do
   :javascript
     var stripePublishableKey = "#{Rails.configuration.stripe[:publishable_key]}"
-
-
   = javascript_include_tag 'donations'
+  %script{ :src => 'https://checkout.stripe.com/checkout.js', :type => 'text/javascript', 'data-cfasync' => 'false' }
```

Comment:
How does this render as the HTML?

IIRC the rocket loader stuff was very explicit about the data-cfasync attribute being before the src attribute. It seems like a weird request but I guess because rocket loader does some kind of naive string parsing rather than understanding the html as html for the sake of performance.

Reply:
it render's with the `data-cfasync` attribute before `src`, like this -> `<script data-cfasync="false" src="https://checkout.stripe.com/checkout.js" type="text/javascript"></script>`.